lib/lapis_lazuli/generators/cucumber/template/features/2_account.feature

```diff
@@ -0,0 +1,38 @@
+@account @all_env
+Feature: User accounts
+  This feature will make sure the user account functionality is working as expected
+  By checking registration, login and logout functionality
+
+  # This is a best practise example. Please note the following
+  # By defining a register, log-in and log-out state, we can easily re-use all these preconditions
+  # All the scenario's aren't completed, but it should be easy to implement it into your own project.
+
+  @account_01 @log_in #@pause # You can add @pause to have a break between every step.
+  Scenario: account_01 - Logging in
+    Given the user is logged out
+    When "test-user" logs in
+    Then the page should display as logged in state
+
+  @account_02 @log_out
+  Scenario: account_02 - Logging out
+    Given "test-user" is logged in
+    When the user clicks on the logout button
+    Then the page should display as logged out state
+
+  @account_03
+  Scenario: account_03 - Opening the registration form
+    Given the user is logged out
+    When the user clicks on the registration button
+    Then the registration form should display
+
+  @account_04
```

Comment:
I get an error when running account04, 05 and 06 on chrome:

When the user completes registration                    # features/step_definitions/account_steps.rb:48
      unknown error: cannot focus element
        (Session info: chrome=62.0.3202.94)
        (Driver info: chromedriver=2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2),platform=Mac OS X 10.13.1 x86_64) (Selenium::WebDriver::Error::UnknownError)

Reply:
Thanks, will look into it.